src/run/run.c

```diff
@@ -80,6 +80,7 @@ static bool arg_quiet = false;
 static bool arg_aggressive_gc = false;
 static char *arg_working_directory = NULL;
 static bool arg_shell = false;
+static const char *arg_job_mode = "fail";
```

Comment:
please copy the string. With `--scope` mode we'll fork off a child, and then try to rename the process via touching `argv[]` which will invalidate all pointers into it. Hence, let's please copy this, the same way as systemctl.c does it. i.e. drop the "const" here, use strdup(), and STATIC_DESTRUCTOR_REGISTER()